electron-1/preload.js

```diff
@@ -0,0 +1,9 @@
+const { ipcRenderer } = require('electron');
+
+// Send synchronous message on "bx" to the main process to get bx
+const bx = ipcRenderer.sendSync('bx');
```

Comment:
How would you have done it with an async call ?
NB: sync IPC calls are blocking the sender process until it receives an answer, so it can be dangerous (deadlocks, throttling, etc.). So when it is possible, async IPC is prefered over sync IPC.

Reply:
I just commit a solution with async messages :)